src/gui/CMakeLists.txt

```diff
@@ -152,7 +152,7 @@ target_sources(owncloudCore PRIVATE
 )
 
 IF( APPLE )
-    find_package(Qt5 COMPONENTS MacExtras)
```

Comment:
maybe we can move all find package stuff to the top of the file 😵‍💫

Reply:
We can do that, but then we have to replicate the platform-specific logic there as well. IMO that could be considered duplicate code.